docs/installation/mac.md

```diff
@@ -61,7 +61,42 @@ redis-cli ping
 
 DEV requires Elasticsearch version 7 or higher.
 
-We recommend installing from archive on Mac. The following directions were
+You have the option of installing Elasticsearch with Homebrew of through an
+archive. We recommend installing from archive on Mac.
+
+Once installed and started you can test if it's up and running correctly by
```

Comment:
I would add this section after the install instructions instead of before. The reason is that people will carry out commands from top to bottom. So they'll install and then will they will test if it's up and running correctly

Reply:
Right! Let me move it down in its own section!